src/main/java/com/autotune/analyzer/services/UpdateRecommendations.java

```diff
@@ -170,8 +167,8 @@ public boolean shouldSkipClass(Class<?> clazz) {
                     .create();
             gsonStr = gsonObj.toJson(recommendationList);
         }
-        if (KruizeDeploymentInfo.logAllHttpReqAndResp)
-            LOGGER.info("Update Recommendation API response: {}", recommendationList);
+        if (KruizeDeploymentInfo.log_http_req_resp)
```

Comment:
Please log error response if not logged already...

in success response , Mention experiment_name , interval_endtime too
will be easy to grep

Reply:
Added experiment_name , interval_endtime in case of success as well